src/types/BooleanSBVR.ts

```diff
@@ -0,0 +1,31 @@
+import * as _ from 'lodash'
+
+const typeFunc = (necessity:string, index:string, defaultValue:string = ' DEFAULT 0') => {
+	return 'INTEGER' + defaultValue + necessity + index
+}
+
+export class BooleanSBVR implements SBVRType<number, boolean> {
```

Comment:
Why did you convert all of the types to classes? There should only ever be one instance so being able to create multiple gives no benefit, also it causes the previous
```
sbvrTypes.boolean.fetchProcessing(...)
```
to become
```
b = new sbvrTypes.BooleanSBVR()
b.fetchProcessing(...)
```
adding extra friction to use it for no gain

(also the fetchProcessing and validate functions are recreated for every instance rather than being shared via the prototype)